components/esp32/Kconfig

```diff
@@ -929,6 +929,24 @@ menu "ESP32-specific"
             This option depends on the CONFIG_FREERTOS_UNICORE option because RTC fast memory
             can be accessed only by PRO_CPU core.
 
+config USE_FIXED_STATIC_RAM_SIZE
+	bool "Use fixed static RAM size"
+	default false
+    help
+	    Without this option, the Dram part of the heap starts right after the .bss sections,
+		within the dram0_0 region. As a result, adding or removing some static variables,
+		indirectly decreases or reduces the heap size.
+		With this option, the Dram part of the heap starts right after the dram0_0 region,
+		where its length is set with FIXED_STATIC_RAM_SIZE
+
+config FIXED_STATIC_RAM_SIZE
+	hex "Fixed Static RAM size"
+	default 0x1E000
+    depends on USE_FIXED_STATIC_RAM_SIZE
+	help
+	    Ram size dedicated for static variables. Should be smaller than 0x2c200.
```

Comment:
This can be achieved with the `range` directive in Kconfig itself (see some other Kconfig files for examples of this).

Reply:
fixed.